src/tsf/ConsoleTSF.cpp

```diff
@@ -3,7 +3,7 @@
 
 #include "precomp.h"
 #include "TfConvArea.h"
-#include "TfEditSes.h"

```

Comment:
Afaict, this was a bug in that a compile would probably have failed on a Case Sensitive (as opposed to Case Preserving) file system :-)